Makefile.rules

```diff
@@ -995,6 +1002,14 @@ testbugfix: all
           ReadGapRoot( "tst/testbugfix.g" );' | $(TESTGAP) | \
             tee `date -u +dev/log/testbugfix2_%Y-%m-%d-%H-%M` )
 
+obj/libgap-test-0001.lo: $(top_srcdir)/tst/testlibgap/0001.c
+	$(QUIET_CC)$(COMPILE) $(DEPFLAGS) $(GAP_CFLAGS) $(CC_EXTRA_FLAGS) $(GAP_CPPFLAGS) -c $< -o $@
```

Comment:
What is `CC_EXTRA_FLAGS`? Is this an accidental left-over, or meant to introduce C flags just for `testlibgap`?

Other than that, with PR #2704 this build rule could go, and the general *.c build rule would take of compiling this C file

Reply:
This is, I believe, an accidental leftover from the history of trying to get libgap into `master`